spec/spec_helper.rb

```diff
@@ -1,31 +1,59 @@
 require 'simplecov'
 SimpleCov.start
 
-require 'manageiq-content'
+if ENV["TRAVIS"] || ENV['CI']
+  require 'coveralls'
+  Coveralls.wear!('rails') { add_filter("/spec/") }
```

Comment:
Is coveralls configured for this repo?

Reply:
It works with `CI` variable and I think it is also for travis to generate coverage report on websites. Only thing I found in this repository about the `coveralls` gem are these 2 lines inside the `spec_helper` file.

Reply:
@bdunne Yes it is, you can see the report as the first comment in the PR: https://github.com/ManageIQ/manageiq-content/pull/524#issuecomment-482217254